app/src/main/res/layout/activity_add_glucose.xml

```diff
@@ -90,6 +90,7 @@
                             android:layout_width="fill_parent"
                             android:layout_height="wrap_content"
                             android:hint="@string/dialog_add_concentration"
+                            android:digits="0123456789.,"
```

Comment:
this allows `,` to show on soft keyboard. as far as i test, it's required. more details at
http://stackoverflow.com/questions/3821539/decimal-separator-comma-with-numberdecimal-inputtype-in-edittext